cfme/tests/configure/test_server_name.py

```diff
@@ -23,27 +24,29 @@ def test_server_name(request, appliance):
 
     view = navigate_to(appliance.server, 'Details')
     old_server_name = view.server.basic_information.appliance_name.read()
+    assert old_server_name != ''  # name should at least be set
 
     @request.addfinalizer
     def _ensure_name_reset():
-        appliance.server.settings.update_basic_information({'appliance_name': old_server_name})
-
-    new_server_name = "{}-TEST".format(old_server_name)
-    appliance.server.settings.update_basic_information({'appliance_name': new_server_name})
-    flash_message = (
-        'Configuration settings saved for {} Server "{} [{}]" in Zone "{}"'.format(
-            appliance.product_name,
-            appliance.server.name,
-            appliance.server.sid,
-            appliance.server.zone.name))
-
-    view.flash.assert_message(flash_message)
+        appliance.rename(old_server_name)
```

Comment:
@mshriver It appears `rename` is a method of an `Appliance` instance but not an `IPAppliance` instance. https://github.com/ManageIQ/integration_tests/blob/976184c0df2ecee9e1baac5132facac5dd8552ef/cfme/utils/appliance/__init__.py#L2745

Do we want this to be method on `IPAppliance`?

Reply:
Agreed, moved.